src/SFA.DAS.AssessorService.Application.Api/StartupConfiguration/Startup.cs

```diff
@@ -57,6 +60,11 @@ public Startup(IHostingEnvironment env, IConfiguration config, ILogger<Startup>
                 UseSandbox = "yes".Equals(config["UseSandboxServices"], StringComparison.InvariantCultureIgnoreCase);
             }
 
+            if (!_env.IsDevelopment())
+            {
+                _azureServiceTokenProvider = new AzureServiceTokenProvider();
```

Comment:
I wonder if theres any implications of generating the bearer token in startup. Can you confirm there are no issues with the app an hour after deployment?

Reply:
@adam230594 I don't think at this point we have actually generated a bearer token. It's only when we call the GetAccessTokenAsync in the ManagedIdentitySqlConnection class that the bearer token is generated.